vyked/decorators/http.py

```diff
@@ -113,10 +116,20 @@ def f(self, *args, **kwargs):
                         'time_taken': int((t2 - t1) * 1000),
                         'process_time_taken': int((tp2-tp1) * 1000),
                         'type': 'http',
-                        'hostname': hostname, 'service_name': service_name
+                        'hostname': hostname,
+                        'service_name': service_name,
+                        'endpoint': func.__name__,
+                        'api_execution_threshold_exceed': False
                     }
-                    logging.getLogger('stats').debug(logd)
-                    _logger.debug('Timeout for %s is %s seconds', func.__name__, api_timeout)
+
+                    method_execution_time = (t2 - t1)
+
+                    if method_execution_time > CONFIG.API_THRESHOLD_TIMEOUT:
```

Comment:
If a request is actually timed out (takes more time than api_timeout) it won't be logged.

Reply:
it is getting logged also, but also add in this logs